usecases/replica/finder.go

```diff
@@ -145,6 +145,7 @@ func (f *Finder) FindUUIDs(ctx context.Context,
 
 	for r := range replyCh {
 		if r.Err != nil {
+			f.logger.WithField("op", "finder.find_uuids").Debug(r.Err)
```

Comment:
do we want to keep this? if yes then can we pass error to `WithError(r.err)` method?